tests/helm/validate_charts.sh

```diff
@@ -85,6 +86,37 @@ validate_custom_endpoint() {
   fi
 }
 
+# Validation test for S3 region configuration
+validate_s3_region() {
+  local chart_dir="$CHARTS_DIR/scality-mountpoint-s3-csi-driver"
+  local custom_region="us-west-2"
+
+  echo "Testing ability to set S3 region..."
+
+  # First check default value
+  echo "Checking default region is set to us-east-1"
+  local result=$(helm template "$chart_dir" --show-only templates/node.yaml 2>&1)
+
+  if ! echo "$result" | grep -q "              value: us-east-1"; then
```

Comment:
nit: you could use something like `grep -Eq "^[[:space:]]*value: us-east-1"` to make sure this doesn't fail if for some reason the number of whitespaces in the beginning changes.
(same for the other grep below)